### PR TITLE
completed extension_converter.py

### PR DESCRIPTION
**Pull request by Phaneron97**
with `extension_converter.py`, the dataset extensions can be converted from .jpg/.jpeg/.bmp to .png.

### CODE REVIEW

**Review by Bastiaan27 — CHANGES_REQUESTED** [review]
test request

File: code/data_analysis/bgr_plotter.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+                    image_rgb = cv2.cvtColor(image, cv2.COLOR_BGR2RGB)
+                    rgb_values.append(image_rgb.reshape(-1, 3))
+
+    return np.vstack(rgb_values)
+
+def plot_rgb_values(rgb_values):
+    """
+    Plot RGB values using matplotlib.
+
+    Parameters:
```
Comment (Bastiaan27) [inline code/data_analysis/bgr_plotter.py:39]: kaas

**Review by Bastiaan27 — APPROVED** [review]
Well done!

File: code/data_analysis/bgr_plotter.py (hunk 1 above)
Comment (Phaneron97) [reply code/data_analysis/bgr_plotter.py:39]: is baas

**Pull request merged**